c_glib/arrow-glib/basic-data-type.cpp

```diff
@@ -69,6 +69,8 @@ G_BEGIN_DECLS
  *
  * #GArrowBinaryDataType is a class for the binary data type.
  *
+ * #GArrowBinaryViewDataType is a class for the binary data type.
```

Comment:
```suggestion
 * #GArrowBinaryViewDataType is a class for the binary view data type.
```
Anyway, could you move this to https://github.com/apache/arrow/blob/40b2fca4742e2692a917755fd8db2939e10fa02d/c_glib/arrow-glib/basic-data-type.cpp#L130 ?

Reply:
Thanks! Fixed.